ha/note.c

```diff
@@ -180,6 +182,24 @@ static void ha_state_accept(struct m0_confc         *confc,
 			if (!ignore_same_state ||
 			    prev_ha_state != obj->co_ha_state)
 				m0_chan_broadcast(&obj->co_ha_chan);
+		} else if ((obj == NULL) &&
+		           (m0_fid_tget(&note->nv_note[i].no_id) == 'r')) {
+				M0_LOG(M0_DEBUG, FID_F" is process FID and not found in conc",
+					FID_P(&note->nv_note[i].no_id));
+				M0_LOG(M0_DEBUG,"Add the entry to conf cache");
+				rc = m0_conf_obj_find(cache, &note->nv_note[i].no_id, 
```

Comment:
Directly call stub_create(cache, id, out); as lookup is already done on line 171 / 169 (old) ?